apps/jobs/views.py

```diff
@@ -1,3 +1,82 @@
+from django.conf import settings
 from django.shortcuts import render
 
-# Create your views here.
+from rest_framework import permissions, status
+from rest_framework.decorators import (api_view,
+                                       authentication_classes,
+                                       permission_classes,
+                                       throttle_classes,)
+from rest_framework.response import Response
+from rest_framework_expiring_authtoken.authentication import (
+    ExpiringTokenAuthentication,)
+from rest_framework.throttling import UserRateThrottle
+
+from accounts.permissions import HasVerifiedEmail
+
+from participants.utils import (
+    get_participant_team_id_of_a_user_for_a_challenge,
+    check_user_participated_in_challenge,)
+
+from challenges.models import (
+    ChallengePhase,
+    Challenge,)
```

Comment:
This is the convention that we should ideally follow and we have been maintaining this throughout the project.